developers_affiliations7.txt

```diff
@@ -1377,8 +1377,6 @@ softwarebloat: softwarebloat!users.noreply.github.com
 	Byte-Code from 2015-10-01 until 2020-02-01
 	iGenius from 2020-02-01 until 2021-03-01
 	Casavo from 2021-03-01
-softwarejimenez: antonjim!cisco.com, softwarejimenez!users.noreply.github.com
-	Cisco
 softwareklinic: keyur.shah!verizon.com
```

Comment:
why removing? If somebody stopped working for `Cisco` then you should just add the end date... or was it because that person was not working for Cisco?

Reply:
Ohh it is the old GitHub login - we should keep it because we are using GitHub archives to track data (they are saved at a time when an activity happens), so we need an old GitHub ID to be able to track this. I will re-add in post-merge.